File: nelio-alves/algoritmo-e-logica-de-programacao/c/exercicios/acima_diagonal.c
#include <stdio.h>

int main() {
    int N, soma;

    printf("Qual a ordem da matriz? ");
    scanf("%d", &N);

    int mat[N][N];

    for (int i = 0; i < N; i++) {
        for (int j = 0; j < N; j++) {
            printf("Elemento [%d,%d]: ", i, j);
            scanf("%d", &mat[i][j]);
        }
    }

    printf("\nMATRIZ DIGITADA:\n");
    for (int i = 0; i < N; i++) {
        for (int j = 0; j < N; j++) {
            printf("%d ", mat[i][j]);
        }
        printf("\n");
    }

    soma = 0;
    for (int i = 0; i < N; i++) {
        for (int j = i + 1; j < N; j++) {
            soma = soma + mat[i][j];
        }
    }

    printf("\nSOMA DOS ELEMENTOS ACIMA DA DIAGONAL PRINCIPAL = %d\n", soma);

    return 0;
}

File: nelio-alves-algoritmo-e-logica-de-programacao/python/exercicios/acima_diagonal.py
N: int; soma: int

N = int(input("Qual a ordem da matriz? "))

mat: [[int]] = [[0 for x in range(N)] for x in range(N)]

for i in range(N):
	for j in range(N):
		mat[i][j] = (int(input(f"Elemento [{i},{j}]: ")))

print("\nMATRIZ DIGITADA:")
for i in range(N):
    for j in range(N):
        print(f"{mat[i][j]} ", end="")
    print()

soma = 0
for i in range(N):
	for j in range(N):
		if i < j:
			soma = soma + mat[i][j]

print(f"\nSOMA DOS ELEMENTOS ACIMA DA DIAGONAL PRINCIPAL = {soma}")